### DISCUSSION TITLE
Ability to include a specific section of another page

### DISCUSSION THREAD

**Discussion by flavius** [requarks/wiki#6119] [Help / Questions] [answered]
Is it possible to include within a page 1 a very specific section (by heading or otherwise) of another page 2, such that edits within page 2 are immediately reflected in page 1?

This is very useful for a knowledge base where I want to present different aspects of the same domain in different ways, different contexts.

In my particular case, it's about IT infrastructure and 1st/2nd level support: the hardware view, the logical view at the application level, a list of problems, their wording and the possible solutions, etc.

**Comment by NGPixel** [comment, marked as answer]
No, follow https://requarks.canny.io/wiki/p/templates-content-transclusion-infoboxes

**Reply by fguisso** [reply]
Do you have any updates on this? Is there a PR with a draft code?
